### DISCUSSION TITLE
Double click the file main.py to start the program

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#2919] [Q&A]
I can double click a python file to start it,  but when using nicegui ,It seems that we must start the program from terminal `python my_nicegui_app.py`

**Comment by falkoschindler** [comment]
Make sure
- that the file is [executable](https://stackoverflow.com/questions/52066176/how-can-i-make-a-script-executable-in-windows) (i.e. has the correct rights) and
- that it starts with a [shebang line](https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take), usually `#!/usr/bin/env python3`.
